package.json

```diff
@@ -43,9 +43,10 @@
     "inline-version": "./scripts/inline-version.js",
     "lint.fix": "yarn lint --fix",
     "lint": "tsc --noEmit && tslint 'src/**/*.ts'",
+    "copy.html-templates": "copyfiles -u 1 \"./src/formatters/html/*.html\" \"./dist/\"",
```

Comment:
Is there any other way to go about this? Is this standard? It feels a little clunky to me, but maybe its fine?

Reply:
> Is there any other way to go about this? Is this standard?

What do you mean? We can place it directly in `prebuild`, no need to have this extra script. I just wanted to make is slightly more readable.

Reply:
I just meant... this feels like an "asset pipeline" sort of thing, I wonder if there are more common ways people handle "not typescript" in typescript libraries than copying bits over. Nevermind :D